lib/cadet_web/controllers/missions_controller.ex

```diff
@@ -27,10 +60,13 @@ defmodule CadetWeb.MissionsController do
       Mission:
         swagger_schema do
           properties do
+            order(:integer, "The order of showing the missions", required: true)
```

Comment:
Just to keep you in the know, we'll probably sort by due date or mission ID on the frontend (perhaps providing a toggle for the user). Still, no harm having this field here for back compatibility with mission data, but just to let you know.